### ISSUE TITLE
help: Using onGlobalSetup from custom nuxt module

### ISSUE THREAD

**Issue by lewebsimple** [nuxt-community/composition-api#406]
**📚 What are you trying to do? Please describe.**
I'm currently developing a [Nuxt module for the Villus GraphQL client](https://github.com/lewebsimple/nuxt-villus) and when trying to add support for @nuxtjs/composition-api I run into the following error (the call stack indicates this is coming from onGlobalSetup):
```
Cannot read property 'add' of undefined
```

**🔍 What have you tried?**
I tried mashing my keyboard with a wooden spoon to make it work but it didn't work :|

**ℹ️ Additional context**
The plugin provided by the Nuxt module is transpiled from TypeScript.
`@nuxtjs/composition-api` is listed both in peerDependencies and devDependencies on the module side of things.
I'm testing the module inside a local project using `yarn link nuxt-villus` but I also tried releasing the module and using it as a normal dependency to no avail.

**Comment by lewebsimple** [comment]
@danielroe many people have said this before, you are a genius!

**Comment by danielroe** [comment]
Should also now be resolved in `v0.22.3`. Thanks for raising the issue! ❤️

**Comment by lewebsimple** [comment]
Getting the following: 
```
Error: Cannot find module '@nuxtjs/composition-api/templates/polyfill.client.js'
```

**Comment by danielroe** [comment]
@lewebsimple Check what Node version you are using.

**Comment by danielroe** [comment]
@lewebsimple (Should be fixed in any case with v0.22.4.)
